portable/GCC/ARM7_AT91SAM7S/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
+                                              unsigned int flag ) /* \arg  flag to be tested */
+    {
+        return( AT91F_PIO_GetInterruptMaskStatus( pPio ) & flag );
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_IsInterruptSet */
+/** \brief Test if PIO Interrupt is Set */
+/**---------------------------------------------------------------------------- */
+    __inline int AT91F_PIO_IsInterruptSet( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                           unsigned int flag ) /* \arg  flag to be tested */
+    {
+        return( AT91F_PIO_GetInterruptStatus( pPio ) & flag );
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_MultiDriverEnable */
+/** \brief Multi Driver Enable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_MultiDriverEnable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                               unsigned int flag ) /* \arg  pio to be enabled */
+    {
         pPio->PIO_MDER = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_MultiDriverDisable
-//* \brief Multi Driver Disable PIO
-//*----------------------------------------------------------------------------
-__inline void AT91F_PIO_MultiDriverDisable(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  pio to be disabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_MultiDriverDisable */
+/** \brief Multi Driver Disable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_MultiDriverDisable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                                unsigned int flag ) /* \arg  pio to be disabled */
+    {
         pPio->PIO_MDDR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetMultiDriverStatus
-//* \brief Return PIO Multi Driver Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetMultiDriverStatus( // \return PIO Multi Driver Status
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetMultiDriverStatus */
+/** \brief Return PIO Multi Driver Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetMultiDriverStatus( /* \return PIO Multi Driver Status */
```

Comment:
Does this need to be on multiple lines?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)